### ISSUE TITLE
Error with SimpleSchema and Collection2

### ISSUE THREAD

**Issue by inromualdo** [JackAdams/meteor-transactions#62]
I can not handle error ...

SimpleSchema.messages({
  "notFound": "La facture n'existe pas",
});

schema= new SimpleSchema({
  b:{
    type:String,
    custom:'function(){
   return "notFound" 
   }
})

I can not handle error notFound, it juste display in console.

**Comment by JackAdams** [comment]
Yes. Sorry. This package's support for SimpleSchema is sketchy at best.

**Comment by inromualdo** [comment]
Okay. So before transaction i must check the schema?
